front/components/poke/plans/form.tsx

```diff
@@ -115,6 +116,16 @@ export const useEditingPlan = () => {
   return { editingPlan, resetEditingPlan, createNewPlan, setEditingPlan };
 };
 
+const BILLING_TYPE_SELECT_CHOICES: Record<
+  FreeBillingType | PaidBillingType,
+  string
+> = {
+  fixed: "Fixed",
+  free: "Free",
+  monthly_active_users: "MAU",
```

Comment:
What's the difference between the two?

Reply:
Ah I see member counts vs active members right? But that's not monthly for seats is it?

Reply:
Yes indeed! I will rename "monthly_active_seats" by "per_seat"